src/common/meta/src/cluster.rs

```diff
@@ -235,7 +242,8 @@ impl From<Role> for i32 {
         match role {
             Role::Datanode => 0,
             Role::Frontend => 1,
-            Role::Metasrv => 2,
+            Role::Flownode => 2,
```

Comment:
Try to keep the values consistent with those in proto: https://github.com/GreptimeTeam/greptime-proto/blob/430eb47c284f421876390b6c0cf49aa3cb4c9cdc/proto/greptime/v1/meta/common.proto#L33